src/hexo/themes/documentation/source/images/results-view-details.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10.41 10.97"><defs><style>.cls-1{fill:#fff;}</style></defs><title>results-view-details</title><path class="cls-1" d="M4.42 6.24H0V4.73h4.42V0H6v4.73h4.42v1.5H6V11H4.42z"/></svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10.41 10.97"><path fill="#fff" d="M4.42 6.24H0V4.73h4.42V0H6v4.73h4.42v1.5H6V11H4.42z"/></svg>
````

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?